aptiblejs/. babelrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
There is a space in this file name - I'm guessing that's not what you want here (i.e. I guess you want `.babelrc`, not `. babelrc`).